spec/scnnr/response_spec.rb

```diff
@@ -121,6 +121,21 @@
         end
       end
 
+      context 'Image Not Found Error' do
+        let(:body) { fixture('image_not_found_error.json').read }
+
+        it do
+          expect { subject }.to raise_error(Scnnr::RecognitionFailed) do |e|
+            expect(e.recognition).not_to be nil
+            expect(e.type).to eq parsed_body['error']['type']
+            expect(e.title).to eq parsed_body['error']['title']
+            expect(e.detail).to eq parsed_body['error']['detail']
+            expect(e.image.status).to eq parsed_body['error']['image']['response']['status']
+            expect(e.image.url).to eq parsed_body['error']['image']['url']
```

Comment:
Thank you for adding the change :) `e.image` being an Image object and having a `url` seem easy to understand :)

I'm not completely sure if an Image's `status` obviously means a response `status`- plus, if we, for some reason, decide to add other stuff, like a response body, it could become a bit more confusing: e.g. `image.body`? (actually some example that included the body existed 😄  https://github.com/NEWROPE/scnnr-consumer/pull/2#discussion_r476357652)

What do you think about also making a `response` object, and completely preserving the nesting? :)

Reply:
Do you mean something like this? https://github.com/NEWROPE/scnnr-ruby/pull/37/commits/64de37eb69f752ec936f3e065ed763e5985882b4
Btw I've rebased the branch due to some conflicts! It's ready to check

Reply:
Ah, thank you for dealing with that. Sorry, it was caused by the Rubocop changes that I pushed...

Yes :) I meant that!